plugins/ruby-foreman-statistics/debian/rules

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+PLUGIN = foreman_statistics
+
+build:
+	cp cache/* /usr/share/foreman/vendor/cache/
+	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
+	cd /usr/share/foreman && ( \
+		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
+		/usr/bin/foreman-ruby /usr/bin/bundle exec rake plugin:apipie:cache[$(PLUGIN)] cache_part=resources \
+		OUT=/var/lib/foreman/public/apipie-cache/plugin/$(PLUGIN) RAILS_ENV=development DATABASE_URL=nulldb://nohost \
+		)
+	[ -e apipie-cache ] || mkdir apipie-cache/
+	cp -rp /var/lib/foreman/public/apipie-cache/plugin/$(PLUGIN) ./apipie-cache/
```

Comment:
Looks like this step is failing for some reason:
```
20:38:43 cp -rp /var/lib/foreman/public/apipie-cache/plugin/foreman_statistics ./apipie-cache/
20:38:43 cp: cannot stat '/var/lib/foreman/public/apipie-cache/plugin/foreman_statistics': No such file or directory
20:38:43 debian/rules:15: recipe for target 'build' failed
20:38:43 make: *** [build] Error 1
20:38:43 dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2

Reply:
this is fixed